assets/multiqc_config.yaml

```diff
@@ -9,3 +9,187 @@ report_section_order:
+        description: |
+            Cis-to-trans ratio is computed as the ratio of long-range cis reads
+            (>20kb) to trans reads plus long-range cis reads. Typically,
+            *cis-to-trans ratio higher than 40% is required*. Percentage of
+            long-range cis reads is the ratio of long-range cis reads to total
+            number of reads. *Minimum 15% is required and 40% or higher suggests
+            a good library*([doi:10.1016/j.cell.2014.11.021](https://pubmed.ncbi.nlm.nih.gov/25497547/)).
+            Convergence is determined as standard deviation of proportions of
+            four read orientations to be <0.002 (Very good) or <0.05 (Good)
+            (See section Proportion of read orientation versus genomic
+            separation in the pairsqc_report file under pairs/QC/). The slope of
+            log10 contact probability vs distance between 10kb ~ 300kb
+            representing TAD is also provided as well. (See section Contact
+            probability versus genomic separation in the pairsqc_report file
+            under pairs/QC/)
+        file_format: "csv"
+        plot_type: 'table'
+        pconfig:
+            id: 'pairs_reads_detailed_summary_table'
+            table_title: 'PAIRS reads detailed summary'
+            namespace: 'PAIRS reads detailed summary'
+            format: '{:.2f}'
+    pairs_reads_proportion:
+        parent_id: 'pairs_qc'
+        section_name: 'Proportion of read orientation versus genomic separation'
+        description: |
+            Contact proportion of reads are shown, stratified by read
+            orientation. Good four cutter samples would show no bias in
+            propotion >3kb(10^3.5).
+        file_format: "json"
+        plot_type: 'scatter'
+        pconfig:
+            id: 'pairs_reads_proportion_vs_distance'
+            title: 'Reads distance vs proportions'
+            xlab: 'distance (kb)'
+            ylab: 'Propotion'
+            ymin: 0
+            ymax: 1
+            marker_size: 3
+            marker_line_colour: '#999999'
+    atac_peaks_qc:
+        parent_id: 'pairs_qc'
+        section_name: 'ATAC R2 reads quality control'
+        description: |
+            *Fraction of reads in peaks (FRiP)* – Fraction of all mapped reads that fall into the called peak regions,
+            i.e. usable reads in significantly enriched peaks divided by all usable reads.
+            In general, FRiP scores correlate positively with the number of regions.
+            ([Landt et al, Genome Research Sept. 2012, 22(9): 1813–1831](https://pubmed.ncbi.nlm.nih.gov/22955991/).
+            *FRiP should be greater than 1%*
+
+            *TSS enrichment score* is a ratio between aggregate distribution of reads centered on TSSs
+            and that flanking the corresponding TSSs.
+            TSS score = the depth of TSS (each 100bp window within 1000 bp each side) / the depth of end flanks (100bp each end).
```

Comment:
Please consider breaking some of these long descriptions out into a `description` and also `helptext`. Then you can use bullet points and more formatting to make it more readable.

See https://multiqc.info/docs/#step-5-create-report-sections - let me know if this doesn't work with Custom Content.

Reply:
Thank you for the help link. I will update long description into a `helptext`.

Reply:
I have trouble in showing the `helptext` in QC report html. I will push the change I made. Please correct me anything I missed.